nautobot/extras/tests/test_api.py

```diff
@@ -1980,8 +1933,8 @@ def test_run_a_job_with_sensitive_variables_and_requires_approval(self, mock_get
         self.assertEqual(
             response.data[0],
             "Unable to run or schedule job: "
-            "This job is flagged as possibly having sensitive variables but is also flagged as requiring approval."
-            "One of these two flags must be removed before this job can be scheduled or run.",
+            "This job is flagged as possibly having sensitive variables but has also approval worklfow definition."
+            "Remove approval workflow definition or set `has_sensitive_variables` to False.",
```

Comment:
Will need a corresponding update if my suggestion above for changes to this message is accepted.